.github/workflows/create-release-tag.yml

```diff
@@ -13,8 +13,8 @@ jobs:
           # BE AWARE --> Updating to a new MAJOR version will delete deprecated versions on a nightly schedule.
```

Comment:
Not related to this PR, but do you know what "a deprecated version" is?

Reply:
No I don't - just copy-pasted :)